src/test/resources/logback-test.xml

```diff
@@ -1,13 +1,11 @@
 <configuration>
   <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
     <encoder>
-      <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+      <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %X %msg%n</pattern>
```

Comment:
Backwards compatibility with logviz is the best reason I can think of to not change it. Not a particularly compelling reason though.

Reply:
I believe `%X` prints the contents of the MDC, but I'm not certain how it's formatted. I typically wrap it in parens because I think it prints key, value pairs.